src/mulsi/processor.py

```diff
@@ -0,0 +1,58 @@
+"""Preprocessing functions for MULSI.
+"""
+
+import einops
+import torch
+from tensordict import TensorDict
+
+
+class DiffClipProcessor:
+    """Differentiable CLIP processor.
+
+    TODO: Temporary solution. Should be inheriting from the processor.
+    """
+
+    def __init__(self, processor):
+        self.processor = processor
+
+    def preprocess(self, image):
```

Comment:
maybe add a doctring here to explain what's happening in the function

Reply:
Sure, it's basically doing what the processor would have done